stdlib/public/Platform/glibc.modulemap.gyb

```diff
@@ -21,6 +21,7 @@
 module SwiftGlibc [system] {
 % if CMAKE_SDK in ["LINUX", "FREEBSD", "CYGWIN"]:
   link "pthread"
+  link "util"
```

Comment:
It's unfortunate we can't make this a directive in the submodule, as this is going to cause everything to link with `util`. However, that is consistent with the current `Glibc` module design (and `Darwin`), so it does seem to fit into the current model.